doc/contributing/maintaining-http.md

```diff
@@ -87,6 +87,8 @@ In order to update Node.js with a new version of llhttp:
 * copy the contents of the `release` directory from the directory you
   checked llhttp out to
   [deps/llhttp](https://github.com/nodejs/node/tree/HEAD/deps/llhttp)
+* update the llhttp version number in the project line in CMakeLists.txt
+  to match that set in include/llhttp.h
```

Comment:
I'm surprised we need to do this. AFAICT this should have been updated in the `make release` step from llhttp: https://github.com/nodejs/llhttp/blob/da60b0bb00b46f0ac58ea48506c6fb6c1b5c112c/Makefile#L54
which should write an updated version of https://github.com/nodejs/llhttp/blob/main/CMakeLists.txt into the `release` directory.

Reply:
ah the TAG stuff looks new https://github.com/nodejs/llhttp/pull/170.

Reply:
That does makes sense. If that will be auto updated then we won't need this update. @mcollina is that correct?

Reply:
@mcollina, ping?

Reply:
@mcollina ping ?

Reply:
Sorry about this, I missed it.

Reply:
I think this is now done automatically. @ShogunPanda can you confirm it too?

Reply:
It should, but I can't be 100% sure as I'm not next to my laptop. Will update tomorrow!

Reply:
I believe the step that need to change is line 86.

From 
`` * run `make release` in the directory that you checked out llhttp. `` 
to  
`` * run `TAG=<version> make release` in the directory that you checked out llhttp. ``
or 
```* run ``TAG=`node -e \"process.stdout.write(require('./package').version)\"` make release`` in the directory that you checked out llhttp. ```

Reply:
The thing is that if llhttp is properly released and downloaded this should not be done at all. Whoever is importing llhttp should just download it from GitHub and copy over.

Let me figure it out.

Reply:
> The thing is that if llhttp is properly released and downloaded this should not be done at all.
 
 If the step is about downloading, which will be the tag `release/vx.y.z`. 
 For example, https://github.com/nodejs/llhttp/releases/tag/release%2Fv6.0.9

Reply:
Exactly. One should not have to download llhttp source and compile locally since we release from GitHub now.

Reply:
Side node: I've just updated llhttp to require the RELEASE variable when running releases command manually, so now `CMakeList.txt`, `llhttp.h` and `package.json` should all report the same version.